frontend/src/components/Tooltip.vue

```diff
@@ -0,0 +1,46 @@
+<template>
```

Comment:
You don't need a tooltip anymore, since the tooltip was related to levels reorder in the table, and this is not gonna happen anymore